### PR TITLE
tanvipanchal's San Diego file addition

### LINKED ISSUE #8: Create a new file for Tanvi San Diego, CA
- [x] Create a branch
- [x] Add commits
- [x] Open a PR
- [x] Collaborate
- [x] Seek approval
- [x] Merge PR

### PR DESCRIPTION
**Pull request by tanvipanchal**
closes #8

### CODE REVIEW

**Comment by SatishMohanV** [comment]
👍

File: tanvipanchal-sandiego.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# San Diego, CA
+
+### Places to Eat
+loving hut mira mesa
```
Comment (SatishMohanV) [inline tanvipanchal-sandiego.md]: will try it out

File: tanvipanchal-sandiego.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+Taste of Himalayas
+
+### Places to Visit
+Point Loma
+La Jolla Beach
```
Comment (SatishMohanV) [inline tanvipanchal-sandiego.md]: will visit

File: tanvipanchal-sandiego.md (hunk 1 above)
Comment (tanvipanchal) [reply tanvipanchal-sandiego.md]: 👍

File: tanvipanchal-sandiego.md (hunk 2 above)
Comment (tanvipanchal) [reply tanvipanchal-sandiego.md]: 👍

**Review by SatishMohanV — APPROVED** [review]
These are great places, Will visit them all.

**Pull request merged**